app/models/user.rb

```diff
@@ -221,13 +224,64 @@ def avatar_url(variant:)
     pic&.url(variant: variant)
   end
 
+  def deleted?
+    deleted_at != nil
+  end
+
+  def soft_delete_relations
+    efficiently_soft_delete_relations
```

Comment:
If you want all `deleted_at` times to line up you could do `efficiently_soft_delete_relations(deleted_at)`.

Reply:
+1